projects/plugins/jetpack/modules/related-posts.php

```diff
@@ -14,19 +14,22 @@
 class Jetpack_RelatedPosts_Module {
```

Comment:
It also complains about `* Additional Search Queries: related, jetpack related posts, related posts for wordpress,` since WordPress isn't capitalized - I'm not sure if it is lower case intentionally for the `Additional Search Queries` field and if I should ignore it.

Reply:
Yea, I would leave it without any capital letters.